Update tag value validation

dgb opened the pull request:
The metrics API has different validation for tag values, namely in that
it supports slashes.

This updates `TAGS_VALUE_REGEX` to match the [restrictions in the HTTP
API documentation](https://www.librato.com/docs/api/#rate-limiting),
along with a test for this case.

dgb committed:
Update tag value validation

The metrics API has different validation for tag values, namely in that
it supports slashes.

This updates `TAGS_VALUE_REGEX` to match the [restrictions in the HTTP
API documentation](https://www.librato.com/docs/api/#rate-limiting),
along with a test for this case.

mbeale commented:
Not sure why the build is failing.  I'll have to investigate.  But the commit lgtm

dgb committed:
Update TAGS_VALUE_REGEX

Updates `TAGS_VALUE_REGEX` to use the current version from the AppOptics
API documentation.

dgb commented:
@petercable Got it. I pulled the pattern from the Librato docs, but I should've done so from the AO docs (I hadn't seen those before this morning 😊 ).

Anyway, PR updated.

prettycoder commented:
The tests are failing with jruby because of timing issues. We can skip some of the tests for jruby or have different timing for jruby. 
How important is jruby for this?

prettycoder commented:
I'm merging this and will fix the jruby test timing issue in master

dgb commented:
Thanks!
